src/Test.UnitTests.Sarif/Writers/SarifLoggerTests.cs

```diff
@@ -348,55 +364,6 @@ public void SarifLogger_WritesFileData()
             sarifLog.Runs[0].Artifacts[0].Hashes["sha-256"].Should().Be("0953D7B3ADA7FED683680D2107EE517A9DBEC2D0AF7594A91F058D104B7A2AEB");
         }
 
-        [Fact]
-        public void SarifLogger_WritesFileContents_EvenWhenLocationUsesUriBaseId()
-        {
-            var sb = new StringBuilder();
-
-            // Create a temporary file whose extension signals that it is textual.
-            // This ensures that the ArtifactContents.Text property, rather than
-            // the Binary property, is populated, so the test of the Text property
-            // at the end will work.
-            using (var tempFile = new TempFile(".txt"))
-            {
-                string tempFilePath = tempFile.Name;
-
-                File.WriteAllText(tempFilePath, "#include \"windows.h\";");
-
-                var run = new Run
-                {
-                    // To get text contents, we also need to specify an encoding that
-                    // Encoding.GetEncoding() will accept.
-                    DefaultEncoding = "UTF-8"
-                };
-
-                var analysisTargets = new List<string>
-                {
-                    tempFilePath
-                };
-
-                using (var textWriter = new StringWriter(sb))
-                {
-                    // Create a logger that inserts artifact contents.
-                    using (_ = new SarifLogger(
-                        textWriter,
-                        run: run,
-                        analysisTargets: analysisTargets,
-                        dataToInsert: OptionallyEmittedData.TextFiles,
-                        levels: new List<FailureLevel> { FailureLevel.Warning, FailureLevel.Error },
-                        kinds: new List<ResultKind> { ResultKind.Fail }))
-                    {
-                    }
```

Comment:
no results are being emitted, with that, no artifacts should be logged. This test was wrong.